tests/esm-path-mapping/built-in-star-path.ts

```diff
@@ -0,0 +1,6 @@
+import * as assert from 'assert';
+
+// Fallback to built-in
+import { stat } from 'fs';
```

Comment:
Attempting to import `assert` is also testing this fallback, since `assert` is a node builtin.

Reply:
Several of the tests have been failing due to this! I wonder if we should add a pre-condition on the other tests so the error is clearer, like:
> failed precondition: assert couldn't be resolved

Then when the tests failed you'll see:
- n x failed precondition: `assert` couldn't be resolved
- 1 x node built-in resolution failed for `assert`